CWE/inventions/culture_inventions.txt

```diff
@@ -629,117 +666,123 @@ social_anthropology = {
 }
 #cold war
 the_atomic_bomb = {
-        limit = { cold_war = 1 }
-        chance = {
-                base = 2
-                modifier = {
-                        factor = 12
-                        OR = { 
+	limit = { cold_war = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
 				tag = USA 
 				tag = RUS 
 			} 
```

Comment:
Should be changed to check for has_country_modifier = un_king1 (No.1 Superpower) OR has_country_modifier = un_king2 (No.2 Superpower)

Reply:
Alright, do you want me to go through and change all instances in the other vanilla inventions to use that as well?

Reply:
Went through and they should all be changed.

Reply:
Thanks!